integration-tests/pom.xml

```diff
@@ -116,7 +116,7 @@
             <cucumber.execution.parallel.config.strategy>fixed</cucumber.execution.parallel.config.strategy>
             <cucumber.execution.parallel.config.fixed.parallelism>${parallel.concurrency.number}</cucumber.execution.parallel.config.fixed.parallelism>
 
-            <sonar.exclusion>**/RunCucumberTest.java</sonar.exclusion>
+            <sonar.exclusion>**/RunCucumberTest.java,**/RunPerformanceCucumberTest.java</sonar.exclusion>
```

Comment:
reading forums, I am not sure `sonar.exclusion` vs `sonar.exclusions`, waiting for official sonar report.

Reply:
ok

Reply:
done, let's see now